### PR TITLE
Verify functionality for multiple GPUs in same cluster [RHODS-3190]

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

Tested locally on live build

### CODE REVIEW

**Comment by lugi0** [comment]
PR Test: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/566

**Review by jgarciao — APPROVED** [review]
lgtm

File: tests/Tests/500__jupyterhub/multiple-gpus.robot [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    [Documentation]    Spawns two servers requesting 1 gpu each, and checks
+    ...    that both can schedule and are scheduled on different nodes.
+    [Tags]    Tier1  Sanity  Resources-2GPUS
+    ...       ODS-1257
+    Spawn Notebook With Arguments  image=${NOTEBOOK_IMAGE}  size=Default  gpus=1
+    ${serial_first} =    Get GPU Serial Number
+    ${node_first} =    Get User Server Node
+    Close Browser
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard  ldap-user5  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
```
Comment (bdattoma) [inline tests/Tests/500__jupyterhub/multiple-gpus.robot:41]: could you use one of the test user in the test-variables file?

Comment (lugi0) [reply tests/Tests/500__jupyterhub/multiple-gpus.robot:41]: I don't see any other users defined: https://github.com/red-hat-data-services/ods-ci/blob/master/test-variables.yml.example#L11

Comment (bdattoma) [reply tests/Tests/500__jupyterhub/multiple-gpus.robot:41]: hm ok, I haven't updated the test-variable example. Please refer to PR #208.

**Review by bdattoma — APPROVED** [review]
lgtm

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**